client/components/Guests.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import plus from '../styles/icons/plus.svg';
+import minus from '../styles/icons/minus.svg';
+
+
+
+const Guests = (props) => {
+  const renderButton = (type, btnIndex) => {
+    //render an icon depending on whether the max guest limit is reached
+    //remove click handler and add disabled class if max guest limit is reached
+
+    let icon = plus;
+    let incrementor = btnIndex % 2 === 0 ? -1 : 1;
+    let clickHandler = () => props.updateGuestCount(type, incrementor);
+    let classNames = 'large-icon';
+    
+    //check if btn index is even and assign icon variable
+    if (btnIndex % 2 === 0) {
+      icon = minus;
+    }
+
+    //check if max guest is reacehed and if current index is for adult/children and if operating on plus icon
+    if ((props.guestCount.adults + props.guestCount.children >= props.maxGuests && (btnIndex === 1 || btnIndex === 3)) 
+      || (props.guestCount[type] === 0 && btnIndex % 2 === 0)
+      || (btnIndex === 0 && props.guestCount.adults === 1)) {
+      classNames += ' disabled';
+      clickHandler = null;
+    }
+
+    return (<img src={icon} className={classNames} onClick={clickHandler}/>);
+  };
+
+  return (
+    <div id="guest-toggle">
+      <div id="adult-guest-toggle">
+        <h3>Adults</h3>
+        {renderButton('adults', 0)}
+        <h3>{props.guestCount.adults}</h3>
+        {renderButton('adults', 1)}
+      </div>
+      <div id="child-guest-toggle">
+        <h3>Children</h3>
```

Comment:
Might be worth adding the age range for children (and infants) - similar to how AirBnb does it